packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor-locked-figures.test.tsx

```diff
@@ -43,59 +51,49 @@ describe("InteractiveGraphEditor locked figures", () => {
             testDependencies,
         );
     });
-    describe("points", () => {
-        test("Calls onChange when a locked point is added", async () => {
+
+    // Basic functionality
+    describe.each`
+        figureType | figureName
+        ${"point"} | ${"Point"}
+        ${"line"}  | ${"Line"}
+    `(`$figureType basics`, ({figureType, figureName}) => {
+        test("Calls onChange when a locked $figureType is added", async () => {
             // Arrange
             const onChangeMock = jest.fn();
 
-            render(
-                <InteractiveGraphEditor
-                    {...mafsProps}
-                    onChange={onChangeMock}
-                />,
-                {
-                    wrapper: RenderStateRoot,
-                },
-            );
+            renderEditor({onChange: onChangeMock});
 
             // Act
             const addLockedFigureButton = screen.getByRole("button", {
                 name: "Add locked figure",
             });
             await userEvent.click(addLockedFigureButton);
-            const addPointButton = screen.getByText("Point");
+            const addPointButton = screen.getByText(figureName);
             await userEvent.click(addPointButton);
 
             // Assert
-            expect(onChangeMock).toBeCalledWith(
-                expect.objectContaining({
-                    lockedFigures: [defaultPoint],
-                }),
-            );
+            expect(onChangeMock).toBeCalledWith({
+                lockedFigures: [getDefaultFigureForType(figureType)],
+            });
         });
 
-        test("Calls onChange when a locked point is removed", async () => {
+        test("Calls onChange when a locked $figureType is removed", async () => {
             // Arrange
             const onChangeMock = jest.fn();
             const confirmSpy = jest.spyOn(window, "confirm").mockImplementation(
                 // Confirm button clicked
                 () => true,
             );
 
-            render(
-                <InteractiveGraphEditor
-                    {...mafsProps}
-                    onChange={onChangeMock}
-                    lockedFigures={[defaultPoint]}
-                />,
-                {
-                    wrapper: RenderStateRoot,
-                },
-            );
+            renderEditor({
+                onChange: onChangeMock,
+                lockedFigures: [getDefaultFigureForType(figureType)],
+            });
 
             // Act
             const deleteButton = screen.getByRole("button", {
-                name: "Delete locked point at 0, 0",
+                name: /Delete*/,
```

Comment:
I don't think this regex will do what you want. It matches "Delet" followed by 0 or more "e"s. You probably just want `/Delete/`.

Reply:
😂 Thank you for pointing that out. I see what you're saying 🤦🏽‍♀️